app/models/user.rb

```diff
@@ -9,7 +9,19 @@ class User < ApplicationRecord
                  key: Rails.application.credentials.encryption_key,
                  encode: true
 
-  validates :first_name, :last_name, presence: true
```

Comment:
Curious, was this to get things up and running during setup, or can we no longer validate for the presence of either of these two name attrs?

Reply:
I believe you can technically have an anonymous google account with no first name and last_name, I don't think we're going to bump against that in oddball though. I think this was just me testing, I'll add those back in